onnx/onnxifi_ext.h

```diff
@@ -7,6 +7,10 @@
 extern "C" {
 #endif
 
+static const int ALL_EXT_FUNCTION_NUMBER = 2;
```

Comment:
please comment these

Reply:
ok..
the "for(int i = 0....)" thing is because, this is C, and for (int i = 0; ...) is not allowed in C...

Reply:
Ohh wow ok got it thanks